src/Apps/MyCollection/Routes/MyCollectionArtwork/Components/MyCollectionArtworkRequestPriceEstimateSection.tsx

```diff
@@ -11,36 +11,48 @@ interface MyCollectionArtworkRequestPriceEstimateSectionProps {
 export const MyCollectionArtworkRequestPriceEstimateSection: React.FC<MyCollectionArtworkRequestPriceEstimateSectionProps> = ({
   artwork,
 }) => {
-  return (
-    <>
-      <Separator my={2} />
-      {artwork.hasPriceEstimateRequest ? (
+  const isP1Artist = artwork.artist?.targetSupply?.isP1
+  const isAlreadySubmitted = artwork.submissionId
+
+  if (artwork.hasPriceEstimateRequest) {
```

Comment:
I've moved all the logic about whether to show the section or not into the component to make it easier to read.